tests/catalog/test_base.py

```diff
@@ -670,14 +670,13 @@ def test_commit_table(catalog: InMemoryCatalog) -> None:
     )
 
     # When
-    response = given_table.catalog._commit_table(  # pylint: disable=W0212
-        CommitTableRequest(
-            identifier=TableIdentifier(namespace=Namespace(given_table._identifier[:-1]), name=given_table._identifier[-1]),
-            updates=[
-                AddSchemaUpdate(schema=new_schema, last_column_id=new_schema.highest_field_id),
-                SetCurrentSchemaUpdate(schema_id=-1),
-            ],
-        )
+    response = given_table.catalog.commit_table(  # pylint: disable=W0212
```

Comment:
```suggestion
    response = given_table.catalog.commit_table(
```
can remove disable pylint